Ability.py

```diff
@@ -1,6 +1,6 @@
 class Ability:
-    def __init__(self, description, HP_delta, HP_delta_duration, tile_match_shape, num_tiles_to_convert, increase_move_count,
-                 berries_to_steal, mana_to_steal, mana_delta, num_mana_to_activate):
+    def __init__(self, description, HP_delta, HP_delta_duration, tile_match_shape, num_tiles_to_convert, move_bonus,
+		move_bonus_duration, berries_to_steal, mana_to_steal, mana_delta, num_mana_to_activate):
```

Comment:
This one is a bit of a stretch. Can you implement default values? 

So for for example it'd change to def __init__(HP_delta = None). 

Then you can get rid of all the 0's and nones. 

I'm explaining this bad
`def greet(name, msg="Good morning!"):
    print("Hello", name + ', ' + msg)

greet("Kate")
greet("Bruce", "How do you do?")`
https://www.programiz.com/python-programming/function-argument
There something like that.

But Flare+ would be

`
Ability('Flare+: Attacks for 25 HP and matches 2 random columns.',
            (0, -25),tile_match_shape = (-1, 1, 2), num_mana_to_activate = 8)
`
And to the end of reducing having to specify the argument name, make sure the most commong arguments are first so that you don't have to specify their names. So num_mana_to_activiate should definitely come way up to the front. 

I realise that this is a big ask so feel free to skip.